docs/cloud/rahti/tutorials/kustomize.md

```diff
@@ -0,0 +1,172 @@
+# Kustomize
+
+[Kustomize](https://kustomize.io) is similar to [Helm](helm.md), both are good for bundling kubernetes elements such as services, deployments, etc...  
+Helm can act as a package manager for kubernetes/oc as well as apt or yum can do for Debian, RedHat.  
+The main difference is Helm uses **Templates** whereas Kustomize uses **Overlays**. Kustomize is also developed by the kubernetes teams, you can build a project using this command:
+
+```sh
+oc kustomize build FOLDER
+```
+
+However some features are missing with the built-in tool, you can install the [tool](https://kubectl.docs.kubernetes.io/installation/kustomize/) separately. The command to build with `kustomize` is:
```

Comment:
Here is the first explicit reference to the fact that kustomize comes integrated with `oc`/`kubectl`. It could make it more clear if this is also mentioned in the first paragraph. Something like:

>  Kustomize is developed by the Kubernetes team and it is built in recent versions of `oc` and `kubectl`.

IMHO it is not bad to repeat information once or twice. First theoretical second practical.

Reply:
Also an example of missing feature, or a link to a list of the missing ones could be nice.